lib/raven/integrations/sidekiq.rb

```diff
@@ -23,6 +23,35 @@ def call(_worker, msg, _queue)
   end
 else
   Sidekiq.configure_server do |config|
-    config.error_handlers << Proc.new {|ex,context| Raven.capture_exception(ex, :extra => {:sidekiq => context}) }
+    config.error_handlers << Proc.new { |ex, context|
+      Raven.capture_exception(ex, :extra => {
+        :sidekiq => filter_context(context),
+      })
+    }
   end
 end
+
+def filter_context(context)
+  case context
+    when Array
+      context.map { |arg| filter_context(arg) }
+    when Hash
+      Hash[context.map { |key, value| filter_context_hash(key, value) }]
+    else
+      context
+    end
+end
+
+def filter_context_hash(key, value)
```

Comment:
I wanted to, but it appears to be private.

https://github.com/rails/rails/blob/master/activejob/lib/active_job/arguments.rb#L130-L135